src/components/ProgressBar/docs/examples.ts

```diff
@@ -0,0 +1,5 @@
+export const importExample = `import { ProgressBar } from 'wix-ui-tpa/ProgressBar';`;
+
+export const example = `
+<ProgressBar />
+`;
```

Comment:
Like the visual test, let's have some examples, with different values, and with/without the indicator